gamestonk_terminal/options/syncretism_view.py

```diff
@@ -297,15 +300,11 @@ def check_presets(preset_dict: dict):
                 error += f"{key} : {value},  Should be [true/false]\n"
 
         elif key == "tickers":
-            if value[0] != '"':
-                error += f'{key} : {value}, ticker should have quotes "AAPL" \n'
-            else:
+            for ticker in value.split(","):
```

Comment:
If the value is a string, isn't ticker gonna be a single char?

like if we do for ticker in "HELLO", we get "H", "E", ...

Reply:
So I checked that  "AAPL" .split(",") gives ['"AAPL"'] which can be eval-ed.  

I tried  cases where I use different combinations and this seems to catch most issues.

Reply:
<img width="285" alt="Captura de ecrã 2021-08-04, às 01 03 34" src="https://user-images.githubusercontent.com/25267873/128101285-eb15b95c-a515-4c46-b9a7-b1b46157f37b.png">

you're right. I thought that if split didn't find the character it would return the string as is. But it still puts it into a list :)

Reply:
I actually did that on accident 🤣